ga4gh/datamodel/__init__.py

```diff
@@ -187,3 +218,21 @@ def _scanDataFiles(self, dataDir, patterns):
             atexit.register(_cleanupHtslibsMess, indexDir)
         if numDataFiles == 0:
             raise exceptions.EmptyDirException(dataDir, patterns)
+
+    def getFileHandle(self, dataFile):
+        if dataFile in self.memo_table:
```

Comment:
I don't think we'll need to access this singleton outside the module, so this module-level variable should be fine. We can look at more sophisticated stuff later if the need arises.